services/motion/builtin/builtin.go

```diff
@@ -25,12 +26,16 @@ import (
 	"go.viam.com/rdk/resource"
 	"go.viam.com/rdk/robot/framesystem"
 	"go.viam.com/rdk/services/motion"
+	"go.viam.com/rdk/services/motion/builtin/state"
 	"go.viam.com/rdk/services/slam"
 	"go.viam.com/rdk/spatialmath"
 	rdkutils "go.viam.com/rdk/utils"
 )
 
-var errUnimplemented = errors.New("unimplemented")
+var (
+	errUnimplemented = errors.New("unimplemented")
+	errClosed        = errors.New("resource (viam:motion:builtin) is closed")
```

Comment:
I'll remove this. Apparently We don't actually need to enforce returning errors when we are closed.

Reply:
Oh, why did this change?

Reply:
it changed b/c I misunderstood the contract between resources & the resource manager. I though (erroneously) that resources needed to gurantee that once their `Close()` method is called, all other resource interface methods (in this case the methods that implement the motion.proto) need to return errors.

After chatting with the netcode team, it turns out that this is actually not a requirement. As a result, I'm going to remove closed tracking, returning error on closed, & this error.

Reply:
Eric had asked me to include that guidance in my talk on bugs and anti-patterns. It would be helpful if the netcode team would send out new guidance, since probably others are still working under that assumption.